content/docs/guides/guacamole.md

```diff
@@ -0,0 +1,164 @@
+---
+# cSpell:ignore guacd, GUACD, guacnetwork
+title: Secure Apache Guacamole
+description: This guide shows you to secure Apache Guacamole behind Pomerium using JWT Claims Headers to support HTTP header authentication.
+keywords: [JWT claims headers, http auth, header auth]
+sidebar_label: Apache Guacamole
+---
+
+# Secure Apache Guacamole
+
+This guide shows you how to secure an instance of [Apache Guacamole](https://guacamole.apache.org/), a clientless, remote desktop gateway that supports VNC, RDP, and SSH protocols.
```

Comment:
Lets clarify the objective at the start &  explicitly state the goal in the introduction, mentioning that the guide will help users set up a secure, authenticated Apache Guacamole instance behind Pomerium.

Reply:
I rewrote the intro based on your feedback.